pkg/flare/envvars.go

```diff
@@ -85,6 +85,12 @@ var allowedEnvvarNames = []string{
 	"DD_APM_SYMDB_DD_URL",
 	"DD_APM_OBFUSCATION_CREDIT_CARDS_ENABLED",
 	"DD_APM_OBFUSCATION_CREDIT_CARDS_LUHN",
+	"DD_APM_OBFUSCATION_MONGODB",
+	"DD_APM_OBFUSCATION_ELASTICSEARCH",
+	"DD_APM_OBFUSCATION_REDIS_ENABLED",
+	"DD_APM_OBFUSCATION_MEMCACHED_ENABLED",
+	"DD_APM_OBFUSCATION_HTTP",
+	"DD_APM_OBFUSCATION_REMOVE_STACK_TRACES",
```

Comment:
nit: maybe it would be good to sort these alphabetically (or at least just the `DD_APM_OBFUSCATION_` ones)